README.markdown

```diff
@@ -1,6 +1,8 @@
 Puppet Labs Spec Helper
 =======================
 
+[![Build Status](https://travis-ci.org/puppetlabs/puppetlabs_spec_helper.svg)](https://travis-ci.org/puppetlabs/puppetlabs_spec_helper)
```

Comment:
Could you actually not add badges to the readme? They end up showing up in all releases but only reflect the status of the master branch and can be misleading.